### DISCUSSION TITLE
Improving Developer Settings Access by Adding a Top-Right Settings Button

### DISCUSSION THREAD

**Discussion by Akshay-Thakare** [ente-io/ente#2374] [Enhancements]
With respect to https://github.com/ente-io/ente/pull/1302 

Clicking on the onboarding screen seven times to access the developer settings seems counterintuitive. Could we replace this with a settings button in the top right corner, similar to Bitwarden?

**Comment by Acha394** [comment]
Hii bhai can you chat with me
